examples/aggregate_merging/src/projectors.rs

```diff
@@ -25,38 +22,32 @@ pub struct CounterProjector;
 // model - in this case, using queries which only update count_a when EventA is received, and count_b
 // when EventB is received, would be sufficient to guarantee soundness.
 #[async_trait]
-impl<T: Clone + Into<ProjectorEvent> + Send + Sync + Serialize + DeserializeOwned> PgProjector<T, CounterError>
```

Comment:
I love seeing all these trait bounds go, I've always feared they were overkill but never actually sat down trying to figure out what the minimal set it.
I'd love to also check the rest of the codebase, if there are other "useless" bounds. For instance, in `Aggregate` there's Send + Sync on anything even if the trait is purely blocking.

Reply:
I think that the problem with aggregate Send + Sync types is that those types are used in `AggregateManager` too that need to have Send + Sync in order to execute its functions.. I'm looking if there's a way to define extra bounds but it seems to me that is not possible..